packages/backend/src/services/SearchService/SearchService.ts

```diff
@@ -54,6 +54,7 @@ export class SearchService extends BaseService {
         user: SessionUser,
         projectUuid: string,
         query: string,
+        source: 'omnibar' | 'ai_search_box',
```

Comment:
should this be source?: 'omnibar' | 'ai_search_box' = 'omnibar'
I am just thinking if it is going to break anything

Reply:
I like that! making it optional and defaulting to omnibar

making the change